### ISSUE TITLE
Consider making p2pool mining the default option

### ISSUE THREAD

**Issue by garth-xmr** [monero-project/monero-gui#3980]
Right now, the default option is still solo mining. For the vast majority of casual miners, perhaps 99.999% of them, solo mining is an unfeasible option. Can we instead have p2pool mining as the default option in the dropdown menu?

**Comment by f2ph7z02sdb** [comment]
Yeah totally agree on that. Also only the CLI has a question for enabling mining when pc is idling. Sad.
This kind of mining in the UI seems quite pointless. I hope it will be more encouraging in the future to mine from the GUI.
